mmpose/datasets/pipelines/topdown_transforms.py

```diff
@@ -284,3 +284,63 @@ def __repr__(self) -> str:
         repr_str += ('(use_dataset_keypoint_weights='
                      f'{self.use_dataset_keypoint_weights})')
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class TopdownGenerateSimCCLabel(BaseTransform):
```

Comment:
I suggest using `TopdownGenerateRegressionLabel` for SimCC. `TopdownGenerateRegressionLabel` should be a general data transform that can support different regression-based methods by giving different encoders.

Reply:
In fact, I'm not sure if it's appropriate to regard SimCC as a heatmap-based method as well. It locates keypoints by doing classification on 1d vectors, both the outputs and targets are 1d vectors.

Reply:
Maybe we can use one transform `TopdownGenerateTarget` for all methods including heatmap-based, regression-based, and SimCC (whatever it belongs to)? The target name (e.g. 'heatmaps', 'reg_labels') can be controlled by a parameter.

And maybe we can use a term like `keypoint_labels` for both the regression methods and SimCC to avoid confusion?